pages/content/amp-dev/events/amp-cs-2019.html

```diff
@@ -337,10 +337,10 @@ <h2>Frequently Asked Questions</h2>
     <div class="ap-o-jumpto-content ap--inline-footer">
       <ul class="ap-o-jumpto-list">
         <li class="ap-m-jumpto-item">
-          <a class="ap-m-jumpto-item-link" href="{{g.doc('/content/amp-dev/events/amp-conf/code-of-conduct.html', locale=doc.locale).url.path}}">Code of Conduct</a>
+          <a class="ap-m-jumpto-item-link" href="{{g.doc('/content/amp-dev/events/amp-conf/code-of-conduct-amp-cs.html', locale=doc.locale).url.path}}">Code of Conduct</a>
```

Comment:
The `href` value points to _/content/amp-dev/events/**amp-conf**/code-of-conduct-amp-cs.html_.
So the **directory** of the `code-of-conduct-amp-cs.html` seems to be wrong here.